Core/include/Acts/Vertexing/IterativeVertexFinder.hpp

```diff
@@ -114,21 +114,25 @@ class IterativeVertexFinder {
     /// decide if a track should be checked for reassignment to other vertices
     double cutOffTrackWeightReassign = 1;
 
-    // Function to extract parameters from InputTrack
+    /// Function to extract parameters from InputTrack
     InputTrack::Extractor extractParameters;
+
+    /// Magnetic field provider
+    std::shared_ptr<MagneticFieldProvider> field;
```

Comment:
Belated comment, but can this not be
```suggestion
    std::shared_ptr<const MagneticFieldProvider> field;
```
(asking since I'm having to const_cast in EIC code...)

Reply:
I believe that's an oversight! Do you want to open a PR? We can also apply a fix to whatever version is convenient for you and make a patch release.

Reply:
Thanks a lot!